src/DeviceInput/deviceInputSystem.ts

```diff
@@ -103,6 +103,24 @@ export class DeviceInputSystem implements IDisposable {
         return device[inputIndex];
     }
 
+    /**
+     * Checks for existing connections to gamepads and registers them, if necessary
+     */
+    public checkForConnectedGamepads() {

```

Comment:
This should be ok to keep public, since we only call `deviceInputSystem.onDeviceConnected` once when we encounter a new device.

Reply:
I've made this private and the function will only be executed when a callback is assigned to onDeviceConnected.  This should still only execute once.